Elemental.Documentation/Samples/Tables/Index.razor

```diff
@@ -2,14 +2,14 @@
 @inherits DemoPage
 
 <AeFlex Vertical class="table-page-container">
-    <DemoSection Path="./Samples/Tables/Examples/PaginatedTable.razor">
+    <DemoSection Path="./Samples/Tables/Examples/PaginatedTableCustomRender.razor">
         <AeFlex class="table-container">
-            <Elemental.Documentation.Samples.Tables.Examples.PaginatedTable />
+            <Elemental.Documentation.Samples.Tables.Examples.PaginatedTableCustomRender />
         </AeFlex>
     </DemoSection>
-    <DemoSection Path="./Samples/Tables/Examples/PaginatedTableCustomRender.razor">
+    <DemoSection Path="./Samples/Tables/Examples/PaginatedTable.razor">
         <AeFlex class="table-container">
-            <Elemental.Documentation.Samples.Tables.Examples.PaginatedTableCustomRender />
+            <Elemental.Documentation.Samples.Tables.Examples.PaginatedTable />
```

Comment:
Let's switch the order so the default (and most likely) use-case is rendered at the top